.github/workflows/publish-dev.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish-Dev
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
this is going to either require develop/feature branches or us to publish release notes on every merge to master to make sure each version is documented

Reply:
It's a dev version so there's no release notes -- only developers should be using these versions. A manual release would be done for majors/minors off the tip of `master` which would have corresponding release notes.